app/views/profiles/_search.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Search for Matches</h1>
+
+<h2>Select proximity to you</h2>
+
+<%= form_tag(search_profile_path(@profile), method: "get") do %>        
+  <%= radio_button_tag(:distance, "5") %>
+  <%= label_tag(:distance_5, "Within 5km", class: 'm-r-2') %>
```

Comment:
I mentioned this to @daljitcheema too, so maybe pair up with him on this. Your labels here won't match their inputs which isn't great for accessibility and capybara testing. Use the `for` attribute to ensure that it matches it's form input